common/rpc_service.go

```diff
@@ -133,6 +133,11 @@ func (s *Service) Method() map[string]*MethodType {
 	return s.methods
 }
 
+// Name will return service name
+func (s *Service) Name() string {
+	return s.name
+}
+
 // RcvrType ...
```

Comment:
pls add comment for this file. thx.

Reply:
@fangyincheng pls add it , thx.

Reply:
> pls add comment for this file. thx.
Notified Mr fang add it ,but not this pr.